recipe/conda_build_config.yaml

```
@@ -4,18 +4,24 @@ triplet:
   - aarch64-conda-linux-gnu
   - s390x-conda-linux-gnu
   - x86_64-w64-mingw32
+  - x86_64-apple-darwin13.4.0
+  - arm64-apple-darwin20.0.0
 ctng_vendor:
   - conda_cos6
   - conda_cos7
   - conda_cos7
   - conda_cos7
```

This would be an opportunity to clean up the old distinction based on distro-version (just a suggestion; not a blocker)

```suggestion
ctng_vendor:
  - conda
  - conda
  - conda
  - conda
```